.gitignore

```diff
@@ -4,7 +4,7 @@
 **/*.npy
 **/*.png
 **/*.gif
-**/*.cpp
+src/pymoo/cython/*.cpp
```

Comment:
Previously this would gitignore a file that was under version control (`../cython/vendor/hypervolume.cpp`)